cli/parse_pdfs.py

```diff
@@ -143,30 +142,31 @@ def parse_file(
                 select_page = select_page_at_random(num_pages)
                 if not select_page:
                     continue
-            # Maybe we should always pass a layout object into the PageParser class.
-            layout_disambiguator = LayoutDisambiguator(
-                image, model, model_threshold_restrictive
+
```

Comment:
I've not added a hyperparameter tuning script for this yet, will do when we have test set.